src/encoded/static/components/patient.js

```diff
@@ -109,7 +141,9 @@ class Patient extends React.Component {
     const surgeryPanelBody = (
       <SurgeryChart chartId="surgery" data={context.surgery} chartTitle="Surgeries Results Over Time"></SurgeryChart>
     );
-
+    const pathPanelBody = (
+      <dl className="key-value">{this.createPathPanel()}</dl>
+    );
```

Comment:
Redundant codes? Didn't show used of this.createPathPanel()

Reply:
Will remove. Thanks!